src/main/java/com/home/MyWizardBot/appConfig/BotConfig.java

```diff
@@ -0,0 +1,36 @@
+package com.home.MyWizardBot.appConfig;
+
+import com.home.MyWizardBot.MyWizardTelegramBot;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.telegram.telegrambots.bots.DefaultBotOptions;
+import org.telegram.telegrambots.meta.ApiContext;
+@Setter
+@Getter
+@Configuration
+@ConfigurationProperties(prefix = "telegrambot")
+public class BotConfig {
+
+    private String webHookPath;
+    private String botUserName;
+    private String botToken;
+
+    //Create bean MyWizardTelegramBot
+    @Bean
+    public MyWizardTelegramBot MyCommunicationTelegramBot() {
+        DefaultBotOptions options = ApiContext.getInstance(DefaultBotOptions.class);
```

Comment:
What is this variable for? If you don't need it, please delete.

Reply:
DefaultBootOptions - required to install a proxy. Since it is not used here, I removed it.